Packs/IllusiveNetworks/Integrations/IllusiveNetworks/README.md

```diff
@@ -0,0 +1,617 @@
+## Overview
+---
+
+The Illusive Attack Management API allows customers to retrieve detected incidents with a forensics timeline, attack surface insights, collect forensics on-demand, and manage a variety of operations with regard to deceptive entities, deception policies, and more.
+This integration was integrated and tested with version 126 of Illusive Networks
+## IllusiveNetworks Playbook
+---
+
+## Use Cases
+---
+
+## Configure IllusiveNetworks on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for IllusiveNetworks.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __Server URL (e.g. https://example.net)__
+    * __API Token__
+    * __Fetch incidents__
+    * __Incident type__
+    * __Trust any certificate (not secure)__
+    * __Use system proxy settings__
+    * __The initial time to fetch from__
+4. Click __Test__ to validate the URLs, token, and connection.
+## Fetched Incidents Data
+---
+

```

Comment:
add here a json example of single fetched incident

Reply:
You mean to put the following json ? 
{
        "CustomFields": null,
        "ShardID": 0,
        "account": "",
        "activated": "0001-01-01T00:00:00Z",
        "attachment": null,
        "autime": 0,
        "canvases": null,
        "category": "",
        "closeNotes": "",
        "closeReason": "",
        "closed": "0001-01-01T00:00:00Z",
        "closingUserId": "",
        "created": "0001-01-01T00:00:00Z",
        "dbotCreatedBy": "",
        "details": "",
        "droppedCount": 0,
        "dueDate": "0001-01-01T00:00:00Z",
        "feedBased": false,
        "hasRole": false,
        "id": "",
        "investigationId": "",
        "isPlayground": false,
        "labels": null,
        "lastJobRunTime": "0001-01-01T00:00:00Z",
        "lastOpen": "0001-01-01T00:00:00Z",
        "linkedCount": 0,
        "linkedIncidents": null,
        "modified": "0001-01-01T00:00:00Z",
        "name": "Illusive Networks Detected ['DECEPTION'] Incident Type",
        "notifyTime": "0001-01-01T00:00:00Z",
        "occurred": "2020-05-04T11:37:10Z",
        "openDuration": 0,
        "owner": "",
        "parent": "",
        "phase": "",
        "playbookId": "",
        "previousRoles": null,
        "rawCategory": "",
        "rawCloseReason": "",
        "rawJSON": "{\"sourceIp\": \"10.90.10.25\", \"sourceOperatingSystem\": null, \"policyName\": null, \"incidentTypes\": [\"DECEPTION\"], \"riskInsights\": {\"stepsToDomainAdmin\": null, \"stepsToCrownJewel\": null}, \"deceptionFamilies\": [\"FAMILY_TYPE_BROWSERS\"], \"lastSeenUser\": null, \"closed\": false, \"unread\": true, \"flagged\": false, \"hasForensics\": false, \"incidentId\": 32, \"incidentTimeUTC\": \"2020-05-04T11:37:10.231Z\", \"sourceHostname\": null, \"userNotes\": null}",
        "rawName": "",
        "rawPhase": "",
        "rawType": "",
        "reason": "",
        "reminder": "0001-01-01T00:00:00Z",
        "roles": null,
        "runStatus": "",
        "severity": 0,
        "sla": 0,
        "sortValues": null,
        "sourceBrand": "",
        "sourceInstance": "",
        "status": 0,
        "type": "",
        "version": 0
    }